.circleci/config.yml

```diff
@@ -33,9 +33,9 @@ jobs:
       - run:
           name: Install Docker Compose
           command: |
-             curl -L https://github.com/docker/compose/releases/download/1.19.0/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
-             chmod +x ~/docker-compose
-             mv ~/docker-compose /usr/local/bin/docker-compose
+            curl -L https://github.com/docker/compose/releases/download/1.19.0/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+            chmod +x ~/docker-compose
+            mv ~/docker-compose /usr/local/bin/docker-compose
```

Comment:
By the way, some Yaml plugin in vscode did this. I'm a bit surprised it was enabled to "formatOnSave" by default. Either way, can't hurt.